roles/mesos-master/files/haproxy.ctmpl

```diff
@@ -0,0 +1,52 @@
+global
+        log /dev/log    local0
+        log /dev/log    local1 notice
+        chroot /var/lib/haproxy
+        maxconn 4000
+        stats socket /run/haproxy/admin.sock mode 660 level admin
+        stats timeout 30s
+        user haproxy
+        group haproxy
+        daemon
+
+defaults
+        log     global
+        mode    http
+        option  httplog
+        option  dontlognull
+        timeout connect 5000
+        timeout client  50000
+        timeout server  50000
+        monitor-uri /index.html
+
+        errorfile 400 /etc/haproxy/errors/400.http
+        errorfile 403 /etc/haproxy/errors/403.http
+        errorfile 408 /etc/haproxy/errors/408.http
+        errorfile 500 /etc/haproxy/errors/500.http
+        errorfile 502 /etc/haproxy/errors/502.http
+        errorfile 503 /etc/haproxy/errors/503.http
+        errorfile 504 /etc/haproxy/errors/504.http
+
+frontend http-in
+        bind *:80
+
+        # Redirect http to https
+        acl is_http hdr(X-Forwarded-Proto) http
+        redirect scheme https code 301 if is_http
+
+        {{ range services }}
+        {{ if and (.Tags.Contains "http") (.Tags.Contains "public") }}
+        acl {{ .ID }}-aclrule hdr(host) -i {{.Name}}.paas.mozilla.community
```

Comment:
It's kind of a pain to mix jinja2 with go templates because of the ``{{ }}`` which are the same. I was wondering if its better to maintain the keys in a Key-Value format in consul and query them in the template.